docs/en/engines/database-engines/replicated.md

```diff
@@ -0,0 +1,50 @@
+
+# Replicated {#replicated}
+
+Supports replication of metadata via DDL log being written to ZooKeeper and executed on all of the replicas for a given database.
+One Clickhouse server can have multiple replicated databases running and updating at the same time.
```

Comment:
It's true, one server can have multiple different replicated databases, but it cannot have multiple replicas of one replicated database (we should notice it).